### PR TITLE
[#994] Part 1: User Profile Enhancements for Intro and MyProfile page 

### PR DESCRIPTION
**Pull request by jhflorey**
This PR addresses issue https://github.com/episphere/connect/issues/994

### CODE REVIEW

File: i18n/en.js [hunk 1]
```diff
@@ -126,6 +127,29 @@ const en = {
...
+            "placeholder": "Re-enter birth year",
+            "data-error-confirmation":"Both the Year of Birth's do not match"
+        },
+        "monthFieldConfirmation": {
+            "data-error-confirmation":"Both the Month of Birth's do not match"
+        },
+        "dayFieldConfirmation": {
+            "data-error-confirmation":"Both the Day of Birth's do not match"
+        },
+        "birthPlaceIntroduction": "We collect this information so that we canmay match any information we collect from other sources, like state health registries, to you.",
```
Comment (bransteitterbr) [inline i18n/en.js:140]: Typo: canmay

File: i18n/es.js [hunk 2]
```diff
@@ -127,6 +128,29 @@
             "title": "Las dos primeras cifras del año de nacimiento deben ser 19",
             "placeholder": "Ingrese su año de nacimiento"
         },
+        "yearListFieldConfirmation": {
+            "placeholder": "Vuelva a ingresar el año",
+            "data-error-confirmation":"Spanish Text"
```
Comment (bransteitterbr) [inline i18n/es.js:133]: Placeholder text still present

File: i18n/es.js [hunk 3]
```diff
@@ -127,6 +128,29 @@
...
+        },
+        "monthFieldConfirmation": {
+            "data-error-confirmation":"Spanish Text"
```
Comment (bransteitterbr) [inline i18n/es.js:136]: Placeholder text still present

File: i18n/en.js (hunk 1 above)
Comment (jhflorey) [reply i18n/en.js]: Solved

**Review by JoeArmani — CHANGES_REQUESTED** [review]
left comments

File: js/components/form.js [hunk 4]
```diff
@@ -90,15 +92,75 @@ export const renderUserProfile = async () => {
                 </div>
             </div>
 
+            <div class="form-group row">
+                <div class="col-md-3">
+                    <select style="margin-left:0px; max-width:188px;" id="UPMonthConfirmation" class="form-control confirmation-field" target="UPMonth" data-i18n="form.monthFieldConfirmation" data-error-confirmation="Both the Month of Birth's do not match">
```
[the remark this answers is not in the document]
Comment (jhflorey) [reply js/components/form.js:97]: @JoeArmani  we are still waiting for the final Spanish text to update all.
https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1699379771757&sharedAccessCode=

File: js/components/form.js [hunk 5]
```diff
@@ -90,15 +92,75 @@ export const renderUserProfile = async () => {
...
+                        <option class="option-dark-mode" value="07" data-i18n="form.monthJuly">July</option>
+                        <option class="option-dark-mode" value="08" data-i18n="form.monthAugust">August</option>
+                        <option class="option-dark-mode" value="09" data-i18n="form.monthSeptember">September</option>
+                        <option class="option-dark-mode" value="10" data-i18n="form.monthOctober">October</option>
+                        <option class="option-dark-mode" value="11" data-i18n="form.monthNovember">November</option>
+                        <option class="option-dark-mode" value="12" data-i18n="form.monthDecember">December</option>
+                    </select>
+                </div>
+                <div class="col-md-4">
+                    <select style="margin-left:0px; max-width:129px;" class="form-control confirmation-field" id="UPDayConfirmation" target="UPDay" data-i18n="form.dayFieldConfirmation" data-error-confirmation="Both the Day of Birth's do not match"></select>
```
[the remark this answers is not in the document]
Comment (jhflorey) [reply js/components/form.js:114]: @JoeArmani  we are still waiting for the final Spanish text to update all.
https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1699379771757&sharedAccessCode=

File: js/components/form.js [hunk 6]
```diff
@@ -90,15 +92,75 @@ export const renderUserProfile = async () => {
...
+                    <input data-i18n="form.yearListField" style="margin-left:0px; max-width:170px;" type="text" class="form-control required-field input-validation" data-error-required='Please select your birth year.' data-validation-pattern="year" data-error-validation="The year you entered is outside of our expected range. Please check your entry." maxlength="4" id="UPYear" title="Birth year, must be in 1900s" Placeholder="Enter birth year">
                     <!--<datalist id="yearsOption"></datalist>-->
                 </div>
             </div>
+            <div class="form-group row">
+                <div class="col-md-4">
+                    <input data-i18n="form.yearListFieldConfirmation" style="margin-left:0px; max-width:170px;" type="text" class="form-control confirmation-field" id="UPYearConfirmation" Placeholder="Re-enter birth year" target="UPYear" data-error-confirmation="Both the Year of Birth's do not match">
```
[the remark this answers is not in the document]
Comment (jhflorey) [reply js/components/form.js:128]: @JoeArmani  we are still waiting for the final Spanish text to update all.
https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1699379771757&sharedAccessCode=

**Comment by jhflorey** [comment]
@JoeArmani @bransteitterbr we will have another PR for updating all the texts. Still waiting for the final specs
https://nih.app.box.com/integrations/officeonline/openOfficeOnline?fileId=1699379771757&sharedAccessCode=

**Review by JoeArmani — APPROVED** [review]

**Review by bransteitterbr — APPROVED** [review]

**Pull request merged**